docs/utilities/data_masking.md

```diff
@@ -425,15 +426,35 @@ You can modify the following values when initializing the `AWSEncryptionSDKProvi
 | **max_messages_encrypted** | `4294967296`          | The maximum number of messages that may be encrypted under a cache entry                      |
 | **max_bytes_encrypted**    | `9223372036854775807` | The maximum number of bytes that may be encrypted under a cache entry                         |
 
-**Changing the default algorithm**
+If required, you have the option to customize the default values when initializing the `AWSEncryptionSDKProvider` class.
+
+=== "aws_encryption_provider_example.py"
+
+```python hl_lines="14-19"
+--8<-- "examples/data_masking/src/aws_encryption_provider_example.py"
+```
 
-The AWS Encryption SDK defaults to using the `AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384` algorithm for encrypting your Data Key. If you want, you have the flexibility to customize and choose a different encryption algorithm.
+**Passing additional SDK arguments**
```

Comment:
```suggestion
##### Passing additional SDK arguments
```

changing to heading so customers can use deep linking in their content, etc. It's not the best visual but what we've got for deep linking in this case.